EventMixing/src/MixBackgroundFrames_module.cc

```diff
@@ -121,28 +137,58 @@ namespace mu2e {
   MixBackgroundFramesDetail::MixBackgroundFramesDetail(const Parameters& pars, art::MixHelper& helper)
     : spm_{ pars().mu2e().products(), helper }
     , pbiTag_{ pars().mu2e().protonBunchIntensityTag() }
-    , meanEventsPerProton_{ pars().mu2e().meanEventsPerProton() }
     , debugLevel_{ pars().mu2e().debugLevel() }
     , engine_{helper.createEngine(art::ServiceHandle<SeedService>()->getSeed())}
     , urbg_{ engine_ }
     , totalBkgCount_(0)
     , skipFactor_{ pars().mu2e().skipFactor() }
     , writeEventIDs_{ pars().mu2e().writeEventIDs() }
+    , simStageEfficiencyTags_{ pars().mu2e().simStageEfficiencyTags() }
+    , meanEventsPerPOTFactors_{ pars().mu2e().meanEventsPerPOTFactors() }
+    , mixingMeanOverride_(false)
   {
     if(writeEventIDs_) {
       helper.produces<art::EventIDSequence>();
     }
+    if (pars().mu2e().meanEventsPerProton(meanEventsPerProton_)) {
+      mixingMeanOverride_ = true;
+    }
```

Comment:
Please disallow specifying both meanEventsPerProton and new style configs.   Inside this if(),  throw an exeception if either of simStageEfficiencyTags or meanEventsPerPOTFactors not empty.

Reply:
Done (commit c6d970e47)